benefits/core/admin/forms.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth.forms import PasswordResetForm
+from django.forms import ValidationError
+
+from benefits.core import recaptcha
+
+
+class BenefitsPasswordResetForm(PasswordResetForm):
+
+    def clean(self):
+        if not recaptcha.verify(self.data):
+            raise ValidationError("reCAPTCHA failed, please try again.")
+        return super().clean()
```

Comment:
thanks for resurfacing the tip. i think the UI here looks quite nice.

not sure how hard it would be, but if we could avoid displaying the 'default' form validation that complains about the forms sole field, that would be ideal yeah?

<img width="440" height="257" alt="Image" src="https://github.com/user-attachments/assets/3b4c3327-a24a-43fb-995e-713e1e0ed1c6" />

Reply:
You mean, not bother showing the error about it being a bad email address if the reCAPTCHA failed?

Reply:
How did you even get that to submit, though? The browser considers `john@compilerl.a` to be a valid email address, but Django doesn't?

Reply:
hah. now that you mention it... my screenshot is an example of a simulated reCAPTCHA failure **and** an invalid email address. 🤦‍♂️ 

when i posted originally i (wrongly) assumed that the reCAPTCHA failure was triggering the field to be considered invalid all on its own.

carry on!